sfn-wdl/sfnwdl_miniwdl_plugin.py

```diff
@@ -96,20 +106,21 @@ def stderr_callback(line):
     except Exception as exn:
         if s3_wd_uri:
             # read the error message to determine status user_errored or pipeline_errored
-            status = "pipeline_errored"
+            status = dict(status="pipeline_errored")
             msg = str(exn)
             if last_stderr_json and "wdl_error_message" in last_stderr_json:
                 msg = last_stderr_json.get("cause", last_stderr_json["wdl_error_message"])
                 if last_stderr_json.get("error", None) == "InvalidInputFileError":
                     status = "user_errored"
                 if "step_description_md" in last_stderr_json:
-                    status["description"] = last_stderr_json["step_description_md"]
```

Comment:
Thanks @kislyuk! my bad here on this line obviously. And it was not on my radar that `aws s3 cp` could generate a standard output message like that (from a custom plugin?). One straightforward follow-up would be to add a `-o output.json` option to miniwdl to have a guaranteed way to get only its output/error JSON.